### PR TITLE
adds effnetb1_cutmixpatch_augmix_robust32_avge4e7_manylayers_324x288 …

### PR DESCRIPTION
**Pull request by mike-ferguson**
Adds model `effnetb1_cutmixpatch_augmix_robust32_avge4e7_manylayers_324x288` (rank 4 on website) to 2.0

### CODE REVIEW

File: brainscore_vision/models/effnetb1_cutmixpatch_augmix_robust32_avge4e7_manylayers_324x288/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+numpy
+torch
+timm @ git+https://github.com/rwightman/pytorch-image-models.git
+albumentations==1.1.0
+ssl
+PIL
+functools
```
Comment (mschrimpf) [inline brainscore_vision/models/effnetb1_cutmixpatch_augmix_robust32_avge4e7_manylayers_324x288/requirements.txt:7]: already part of standard python
```suggestion
```

**Review by mschrimpf — APPROVED** [review]

**Comment by samwinebrake** [comment]
@mike-ferguson looks like min python version needed for timm is 3.8

**Comment by mike-ferguson** [comment]
PR will not pass until Brain-Score is updated to support Python >3.7, On hold for now

**Comment by mike-ferguson** [comment]
Closed due to:

1) Needing reproduction on Sanghavi Benchmark 
2) Dependent on Python >3.7

**Pull request closed without merging**